prettier-js.el

```diff
@@ -138,7 +90,7 @@ a `before-save-hook'."
       ;; Convert the prettier stderr to something understood by the compilation mode.
       (goto-char (point-min))
       (insert "prettier errors:\n")
-      (while (search-forward-regexp (regexp-quote tmpfile) nil t)
+      (while (search-forward-regexp "^stdin" nil t)
```

Comment:
This also fixes a bug--the error contained `stdin` rather than the tmpfile name, so now the error display actually shows the real file name as intended.